google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutation.java

```diff
@@ -34,60 +35,82 @@
 public final class RowMutation implements MutationApi<RowMutation>, Serializable {
   private static final long serialVersionUID = 6529002234913236318L;
 
-  private final String tableId;
+  private final TargetId targetId;
   private final ByteString key;
   private final Mutation mutation;
 
-  private RowMutation(String tableId, ByteString key, Mutation mutation) {
-    this.tableId = tableId;
+  private RowMutation(TargetId targetId, ByteString key, Mutation mutation) {
+    Preconditions.checkNotNull(targetId, "target id can't be null.");
+
+    this.targetId = targetId;
     this.key = key;
     this.mutation = mutation;
   }
 
-  /** Creates a new instance of the mutation builder. */
-  public static RowMutation create(@Nonnull String tableId, @Nonnull String key) {
-    return create(tableId, ByteString.copyFromUtf8(key));
+  /** @deprecated Please use {@link RowMutation#create(TargetId, String)} instead. */
+  public static RowMutation create(String tableId, String key) {
+    return create(TableId.of(tableId), ByteString.copyFromUtf8(key));
```

Comment:
nit: here you can just do
```
return create(tableId, ByteString.copyFromUtf8(key));
```
since there's another create function that converts String to TableId